libraries/matchers.rb

```diff
@@ -16,7 +16,7 @@
   oneview_resources = {
     oneview_resource:                   standard_actions,
     oneview_enclosure:                  [:add, :remove],
-    oneview_enclosure_group:            standard_actions,
+    oneview_enclosure_group:            [:create, :create_if_missing, :delete, :set_script],
```

Comment:
I think it would be better to describe it as: `standard_actions + [:set_script]`